server/ce2e/iframe_test.go

```diff
@@ -0,0 +1,94 @@
+package ce2e
```

Comment:
This is another example of tests migrated from mocks to using the containerized approach.